build_tools/update_shape_lib.sh

```diff
@@ -9,26 +9,27 @@
 # For more information on supporting custom operators, see:
 #   ${TORCH_MLIR}/python/torch_mlir/_torch_mlir_custom_op_example/README.md
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
why this change ?

Reply:
`set -u` throws an unbound variable error if a referenced variable isn't set. When a user doesn't want to add a custom operator `TORCH_MLIR_EXT_PYTHONPATH` and `TORCH_MLIR_EXT_MODULES` variables aren't initialized thus relaxing the need for it to be set.

Reply:
I think we should set it to default if not set.

Reply:
like https://github.com/llvm/torch-mlir/blob/9cf0b6e8ff4ed2400175475c7ca7694688cf17f1/build_tools/python_deploy/build_linux_packages.sh#L43